UI/js-src/lsmb/menus/Cache.js

```diff
@@ -0,0 +1,41 @@
+define(["dojo/_base/lang","dojo/when" /*=====, "dojo/_base/declare", "dojo/store/api/Store" =====*/],

```

Comment:
Because we wanted to cache each child without having to pull the whole tree each time and remove any possibility for add of remove, which are present in the standard Dojo store cache.